src/main/java/org/gridsuite/modification/server/ModificationType.java

```diff
@@ -23,6 +23,7 @@ public enum ModificationType {
     GENERATOR_CREATION(PreloadingStrategy.NONE),
     GENERATOR_MODIFICATION(PreloadingStrategy.NONE),
     EQUIPMENT_DELETION(PreloadingStrategy.NONE),
+    BY_FILTER_DELETION(PreloadingStrategy.NONE),
```

Comment:
EQUIPMENT_BY_FILTER_DELETION(PreloadingStrategy.COLLECTION) ?

Reply:
@SlimaneAmar I indeed missed that. I understand that we want to use PreloadingStrategy.COLLECTION to load and cache the whole collection from network-store and improve loading efficiency. What's the difference with PreloadingStrategy.ALL_COLLECTIONS_NEEDED_FOR_BUS_VIEW ?
In the case of PreloadingStrategy.NONE we only cache the loaded item, correct ?